anti.md

```diff
@@ -5,40 +5,78 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+Doing laundry
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Doing laundry is such a pain.  No one enjoys doing it.  It takes a lot of work
+sorting everything by material and color, and then actually washing, drying,
+folding and then putting away your clothes is a pain (not to mention things that
+have to be ironed and/or hung up).  This helps people who don't like doing 
+laundry, but not really.  
+
+You can think of how to do laundry as one set of rules that could be globally
+applied to all laundry.  It doesn't take writing a new program to do laundry
+in a different home - just a robot that knows the rules.
```

Comment:
I feel like there are some cases when your program wouldn't compile on a different laundry machine, like if it doesn't have some settings that your program makes use of.